packages/artifacts/src/generate-artifacts.ts

```diff
@@ -2,19 +2,15 @@ import {writeFileSync} from "fs"
 
 import {Language, lioncore, lioncoreBuiltins, serializeLanguages, serializeNodes} from "@lionweb/core"
 import {
-    languageAsText,
     generateMermaidForLanguage,
     generatePlantUmlForLanguage,
     GenerationOptions,
+    languageAsText,
     tsTypesForLanguage,
     writeJsonAsFile
 } from "@lionweb/utilities"
-import {libraryLanguage} from "@lionweb/test/dist/languages/library.js"
-import {multiLanguage} from "@lionweb/test/dist/languages/multi.js"
-import {languageWithEnum} from "@lionweb/test/dist/languages/with-enum.js"
-import {libraryExtractionFacade, libraryModel} from "@lionweb/test/dist/instances/library.js"
-import {multiExtractionFacade, multiModel} from "@lionweb/test/dist/instances/multi.js"
-import {shapesLanguage} from "@lionweb/test/dist/languages/shapes.js"
+import {languageWithEnum, libraryLanguage, multiLanguage, shapesLanguage} from "@lionweb/test/languages"
+import {libraryExtractionFacade, libraryModel, multiExtractionFacade, multiModel} from "@lionweb/test/instances"
```

Comment:
So why not just a normal export like all other packages?

Reply:
No, it's a bit more: `... from "@lionweb/test/dist/languages/<language>.js"` &rarr; `... from "@lionweb/test/languages"`. And I wanted to test NPM submodules out because I also want to use them for the `core` package.

Reply:
If we just use the regular export we would get `import ... from "@lionweb/test"`.

Reply:
True, but we'd also be exposing `languages` and `instances` on the top-level...which I'm doing now as well, so I'll have to try out what happens if we don't do that.

Reply:
It seems that the `exports` section in the `package.json` does prohibit that `languages` and `instances` are exposed on the top-level – which is exactly what I want.

Reply:
Why is that important to you?